todo/components/Button/index.tsx

```diff
@@ -0,0 +1,15 @@
+import { FC, ReactNode, MouseEvent } from 'react';
+
+type ButtonProps = {
+    onClick: (e: MouseEvent) => void
+    disabled?: boolean
+    className: string
+    children: ReactNode
+}
+
+export const Button: FC<ButtonProps> = ({
+    onClick,
+    disabled,
+    className,
+    children,
+}) => <button onClick={(e: MouseEvent) => onClick && onClick(e)} className={className} disabled={disabled}>{children}</button>;
```

Comment:
`onClick && `は必要ないかな〜
これやってる理由は、onClickが`undefined`のケースがあるから、それを判定しするために`&&`している

```suggestion
}) => <button onClick={(e: MouseEvent) => onClick(e)} className={className} disabled={disabled}>{children}</button>;
```